README.md

```diff
@@ -966,59 +970,81 @@ You can use iMessage with either a [phone number or an email](https://support.ap
 
-Subscribe to updates at [The Safe Mac](http://www.thesafemac.com/) and [Malwarebytes Blog](https://blog.malwarebytes.com/) for current Mac security news.
+Otherwise, get programs from trusted sources like directly from the developer's website or GitHub. Always make sure that your browser/terminal is using HTTPS when downloading any program.
 
-To scan an application with multiple AV products and examine its behavior, upload it to [VirusTotal](https://www.virustotal.com/#/home/upload).
+You should also avoid programs that ask for lots of permissions and third party closed source programs.
 
-Also check out [Hacking Team](https://www.schneier.com/blog/archives/2015/07/hacking_team_is.html) malware for macOS: [root installation for MacOS](https://github.com/hackedteam/vector-macos-root), [Support driver for Mac Agent](https://github.com/hackedteam/driver-macos) and [RCS Agent for Mac](https://github.com/hackedteam/core-macos), which is a good example of advanced malware with capabilities to hide from userland (e.g., `ps`, `ls`). For more, see [A Brief Analysis of an RCS Implant Installer](https://objective-see.com/blog/blog_0x0D.html) and [reverse.put.as](https://reverse.put.as/2016/02/29/the-italian-morons-are-back-what-are-they-up-to-this-time/)
+## App Sandbox
 
-# System Integrity Protection
+Check if a program uses the [App Sandbox](https://developer.apple.com/documentation/security/app_sandbox/protecting_user_data_with_app_sandbox) before running it by running the following command:
+
+```console
+codesign -dvvv --entitlements - <path to your app>
+```
 
-To verify SIP is enabled, use the command `csrutil status`, which should return: `System Integrity Protection status: enabled.` Otherwise, [enable SIP](https://developer.apple.com/library/content/documentation/Security/Conceptual/System_Integrity_Protection_Guide/ConfiguringSystemIntegrityProtection/ConfiguringSystemIntegrityProtection.html) through Recovery Mode.
+If the App Sandbox is enabled, you will see
 
-# Gatekeeper and XProtect
+```console
+    [Key] com.apple.security.app-sandbox
+    [Value]
+        [Bool] true
+```
 
-**Gatekeeper** and the **quarantine** system try to prevent unsigned or "bad" programs and files from running and opening.
+Alternatively, you can check while the app is running by opening Activity Monitor and adding the "Sandbox" column.
 
-**XProtect** prevents the execution of known bad files and outdated plugin versions, but does nothing to cleanup or stop existing malware.
+All App Store apps are required to use the App Sandbox.
 
-See also [Gatekeeper, XProtect and the Quarantine attribute](https://ilostmynotes.blogspot.com/2012/06/gatekeeper-xprotect-and-quarantine.html).
+**Note:** Browsers like Google Chrome use their own strong sandbox so they don't use the App Sandbox. This is fine, you mainly need to worry about other types of programs not being sandboxed.
```

Comment:
Why is it "fine"? What is this assertion based on? Perhaps link the chromium white paper on sandboxing here?

Reply:
Also that sentence is something dangerous. People will think other browser (because of „like chrome“) are safe too which isn’t true.

Reply:
added a link to their sandbox documentation